java/test/behaviour/connection/ConnectionStepsBase.java

```diff
@@ -81,8 +87,11 @@ void before() {
     void after() {
         cleanupTransactions();
 
-        driver = createTypeDBDriver(TypeDB.DEFAULT_ADDRESS);
-        driver.databases().all().forEach(Database::delete);
+        driver = createDefaultTypeDBDriver();
+        // TODO: A temporary filter hack before we hide the system db
```

Comment:
Will want to remove it when "system" is fully hidden.

Reply:
@lolski  what happens if the user accidentally tries to delete system db - hopefully an exception right? I wonder how we handled this in 2.x BDD